testing/constraints-3.6.txt

```
@@ -0,0 +1,11 @@
+# This constraints file is used to check that lower bounds
+# are correct in setup.py
+# List *all* library dependencies and extras in this file.
+# Pin the version to the lower bound.
+#
+# e.g., if setup.py has "foo >= 1.14.0, < 2.0.0dev",
+# Then this file should have foo==1.14.0
+google-api-core==1.27.0
```

Needs to pin `google-auth` to fix coverage failures
```suggestion
google-api-core==1.27.0
google-auth==1.24.0  # TODO: remove when google-auth >= 1.25.0 is required transitively through google-api-core
```